lib/Test/Nginx/Util.pm

```diff
@@ -918,6 +918,12 @@ http {
     default_type text/plain;
     keepalive_timeout  68;
 
+    client_body_temp_path "$ServRoot/client_body_temp";
+    proxy_temp_path "$ServRoot/proxy_temp";
+    fastcgi_temp_path "$ServRoot/fastcgi_temp";
+    scgi_temp_path "$ServRoot/scgi_temp";
+    uwsgi_temp_path "$ServRoot/uwsgi_temp";
```

Comment:
Another thing is that you are assuming `ngx_proxy`, `ngx_fastcgi`, `ngx_scgi`, and `ngx_uwsgi` modules are always enabled in the nginx used in testing, which is apparently not the case. Sometimes we explicitly want to test a 3rd-party module when some or all of these standard nginx modules are absent.